src/main/java/core/basesyntax/service/StoreServiceImpl.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.operations.Operations;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class StoreServiceImpl implements StoreService {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String TITLE = "fruit,quantity";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final int INDEX_OF_DATA_START = 1;
+    private final OperationStrategy operationStrategy;
+    private final ProductDao productDao;
+
+    public StoreServiceImpl(OperationStrategy operationStrategy, ProductDao productDao) {
+        this.operationStrategy = operationStrategy;
+        this.productDao = productDao;
+    }
+
+    @Override
+    public void addToStorage(List<String> dataFromFile) {
+        for (int i = INDEX_OF_DATA_START; i < dataFromFile.size(); i++) {
+            String[] data = dataFromFile.get(i).split(CSV_SEPARATOR);
+            Product product = new Product(data[FRUIT_INDEX]);
+            int oldValue = productDao.get(product);
+
+            String operationType = data[OPERATION_INDEX].toUpperCase();
```

Comment:
Ideally, we move validation into separate service and change enums to full names with short versions as a string values